src/rnp/rnp.cpp

```diff
@@ -906,6 +919,15 @@ main(int argc, char **argv)
         }
     }
 
+    switch (rnp_cfg_getint(&cfg, CFG_COMMAND)) {
+    case CMD_HELP:
+    case CMD_VERSION:
+        rnp_cmd(&cfg, &rnp);
+        ret = EXIT_SUCCESS;
```

Comment:
Should this be conditional based on the return of `rnp_cmd`?

Reply:
The same as previous. Will update the PR shortly.